app/pandas.ipynb

```diff
@@ -145,18 +156,73 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "persons[persons.Age > 18]",
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-05T09:17:50.594551Z",
-     "start_time": "2023-07-05T09:17:50.580721Z"
+     "end_time": "2024-11-16T09:23:39.018130Z",
+     "start_time": "2024-11-16T09:23:39.009905Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "    Name  Age         City  Salary\n",
+       "1  Emily   28  Los Angeles   60000\n",
+       "2   Ryan   22      Chicago  450000"
```

Comment:
There seems to be a typo in the output for the 'Salary' column. The value for Ryan should be 45000, not 450000. Please correct this to ensure the data is accurate.